package.json

```diff
@@ -7,13 +7,16 @@
     "deploy-staging": "sh scripts/deploy-staging.sh",
     "deploy-storybook": "storybook-to-ghpages",
     "deploy": "sh scripts/deploy.sh",
+    "generate-types": "apollo codegen:generate --target typescript --outputFlat src/v2/apollo/types",
+    "generate-types:watch": "yarn generate-types --watch",
     "jest": "jest",
     "lint": "eslint --ext .js,.jsx,.ts,.tsx ./src --fix",
     "mocha": "sh scripts/mocha.sh",
     "predeploy": "sh scripts/predeploy.sh",
     "restart-dyno": "node bin/restart",
     "start-dev": "node src/index.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
+    "sync-schema": "babel-node scripts/syncSchema.js",
```

Comment:
Noticed y'all have a `schema.graphql` and a `schema.json` file in `v2/apollo`. Figured this would be a handy script to pull updates in.